packages/nc-gui/components/smartsheet/Pagination.vue

```diff
@@ -93,15 +94,17 @@ const isRTLLanguage = computed(() => isRtlLang(locale.value as keyof typeof Lang
         </a-input>
       </div>
     </template>
-
-    <div class="flex-1 text-right">
-      <span
-        v-if="alignCountOnRight && count !== null && count !== Infinity"
-        class="caption nc-grid-row-count mr-2.5 text-gray-500 text-xs"
-        data-testid="grid-pagination"
-      >
-        {{ count }} {{ customLabel ? customLabel : count !== 1 ? $t('objects.records') : $t('objects.record') }}
-      </span>
+    <div class="flex-1 flex justify-end items-center">
+      <GeneralApiTiming v-if="isEeUI && props.showApiTiming" class="m-1" />
+      <div class="text-right">
+        <span
+          v-if="alignCountOnRight && count !== null && count !== Infinity"
```

Comment:
just wondering if `count` can be `null` here as it should be set to `Infinity` if it is `null`

Reply:
You are right, it will never be `null` since we are returning `Infinity` on null. Good catch! :+1: